elections/committee-conflict-of-interest.md

```diff
@@ -0,0 +1,15 @@
+# Committee Conflict of Interest
+
+At time of writing, the only known conflict of interest for serving on
+committees in the Kubernetes Community exists with the
+[Steering Committee](/committee-steering/README.md) and
+the [Code of Conduct Committee](/committee-code-of-conduct/README.md).
+
+**_Kubernetes Community members cannot serve on both committees simultaneously._**
+
+Upon appointment, the (Steering Committee) election committee will reach out to
```

Comment:
Note here, I've used "Upon appointment...", instead of "Upon nomination...", but I'm happy to update it to something more fitting.

Similar to [maximal representation](https://github.com/kubernetes/steering/blob/c65bd4e586be59d12466b2bd6c19905240a97f0a/elections.md#maximal-representation), the conflict is not triggered until a condition is met i.e., it is determined that the candidate would be serving on both Steering and CoCC.

That condition cannot be met until an election has been completed.

That is to say, if a community member opts to run for:

- either committee while being a standing member of the other, this is allowed
- both committees and is not a standing member of either, that is also allowed

Reply:
for the elections, wouldn't it be better to reach out to the candidate BEFORE the election results are announced, rather than afterwards?

Reply:
So, running for both committees is a problem as demonstrated by the current elections.   This puts us in a situation where the results of the election to conclude first need to be put on hold until the second election results are in.  I really think that we need to prohibit running for both positions at once.

Reply:
+1, to update this to `Upon nomination`.

If a candidate is running for both commitees, IMO it is fair to ask them before the election to keep things transparent for everyone.